docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -63,3 +63,15 @@ For example:
 
 The same applies to `slotProps` and `componentsProps`.
 :::
+
+### Replace `defaultCalendarMonth` with `referenceDate`
+
+The `defaultCalendarMonth` has been removed in favor of the more flexible `referenceDate` prop.
+It was available in `DateCalendar` and all the pickers using it for date editing.
+
+The new `referenceDate` prop is not limited to the default month, learn more on this prop on [the `DateCalendar` doc](/x/react-date-pickers/date-calendar/#choose-the-initial-year-month) or [the `referenceDate` doc](/x/react-date-pickers/base-concepts/#reference-date-when-no-value-is-defined).
```

Comment:
```suggestion
:::warning
The new `referenceDate` prop is not limited to the default month.
It will also impact year, day, and time.

Learn more on this prop on [the `DateCalendar` doc](/x/react-date-pickers/date-calendar/#choose-the-initial-year-month) or [the `referenceDate` doc](/x/react-date-pickers/base-concepts/#reference-date-when-no-value-is-defined).
:::
```

Not sure about the warning. But I would recomand to first point explicitly what could be modified by this change